packages/cli/package.json

```diff
@@ -0,0 +1,28 @@
+{
+    "name": "@synthql/cli",
+    "version": "0.0.1",
```

Comment:
use the same version number as the other packages

Reply:
We should try to make sure all packages in the monorepo all have the same version. This makes it easier for users to figure out which package versions are compatible with each other: "just make sure they are all using the same version"